clarity/data/HOA_tools_cec2.py

```diff
@@ -145,74 +145,80 @@ def P(i, a, b, el, r):
 
 
 @njit
-def U(m, n, el, r):
+def U(degree, n, order, rotation_matrices):
     """U coefficient initialiser for rotation matrix calculation.
 
     Args:
-        m (int): degree
```

Comment:
the use of degree here is not as in subdivision of a circle, but the parameters of a spherical harmonic decomposition.

The variable names for the code called by the rotation matrix generation are defined here:

 'Rotation Matrices for Real Spherical Harmonics. Direct Determination
    by Recursion' Joseph Ivanic and Klaus Ruedenberg J. Phys. Chem. 1996, 100, 15,
    6342–6347

and 

'Forier Acoustics' E. G. Williams https://doi.org/10.1016/B978-0-12-753960-7.X5000-1

in this cale, m refers ro the upper parameter of spherical harmonic component Y and n the lower.

Reply:
Thanks @jwillbailey that maths there is a bit beyond me, I've changed the text to reflect what you've written and modified docstrings to correctly cite the articles in Restructured Text. In doing so I looked up the DOI for the papers and found there was a [correction to the Ivanic and Klaus paper](https://pubs.acs.org/doi/10.1021/jp9833350), not sure if it would have any bearing on what has been implemented here. :shrug: